test/tests/optimizationreporter/objective_gradient_minimize/bc_load_linearFunction/master.i

```diff
@@ -22,21 +22,35 @@
   parameter_names = 'p1 p2'
   num_values = '1 1'
   measurement_points = '0.2 0.2 0
-            0.8 0.6 0
-            0.2 1.4 0
-            0.8 1.8 0'
+                        0.8 0.6 0
+                        0.2 1.4 0
+                        0.8 1.8 0'
   measurement_values = '207 204 185 125'
 []
 
 [Executioner]
   type = Optimize
-  tao_solver = taolmvm
-  petsc_options_iname = '-tao_gatol'#
-  petsc_options_value = '1e-4'
+  # tao_solver = taobncg
```

Comment:
Remove all commented lines.

Reply:
We will do that as a separate commit later.

Reply:
I've been commenting out solver settings in a lot of tests so that I can easily switch between the different optimization parameters.  I'll add an issue to delete these.